source/playbooks/notifications-and-updates.rst

```diff
@@ -1,11 +1,20 @@
 Notifications and Updates
 =========================
 
-To help you keep track of your runs, tasks, and statuses, a daily digest is sent via Direct Message in Mattermost.
+There are multiple ways to receive updates and notifications.
+
+Follow runs
```

Comment:
@stephenvanhemmen, what happens when a run that you're following is finished? Do you get "unsubscribed"? Can that run be restarted? When would following a playbook be better than following a specific run, and vice versa?

Reply:
When a run you're following is finished, you will receive a `Run finished` notification.
<img width="673" alt="Screen Shot 2021-12-13 at 12 45 23 PM" src="https://user-images.githubusercontent.com/67066684/145862166-4cea3617-3053-47fa-aa73-cc4b256d26a2.png">
 
 You will still be following the run (as seen on the `Run Details` page) but should not receive further notifications except if the retrospective is published/republished, then you'll receive a `Retrospective` notification. 
 
Following a playbook would be ideal for users who care about all instances of a certain procedure (usually managers/decision-makers), for example, Tao would likely be auto-following all runs of the Feature Swimlane playbook. Following a specific run would be ideal for users who only care about one instance of a procedure, for example, a community translator would only want to follow the Playbooks translation feature instead of all of our feature runs.